src/common/hooks/money/useFormatMoney.ts

```diff
@@ -30,7 +31,7 @@ export function useFormatMoney() {
     const currency = resolveCurrency(currentCurrencyId);
 
     if (country && currency) {
-      return Number.formatMoney(value, currency, country, {
+      return Number.formatMoney(isNaN(value) ? 0 : value, currency, country, {
```

Comment:
Is there a specific reason for not using JS built-in isNaN?

Reply:
@beganovich Actually, there is no big reason to do that, except that we do not need to cast to a number when passing it to the `isNaN` function from the `Lodash` package, we can pass string also. But yeah, I switched it to the `built-in` isNaN function to check this state. Let me know your thoughts.